walkthroughs/howto-external-traffic/README.md

```diff
@@ -0,0 +1,181 @@
+# Configuring TLS with AWS Certificate Manager
+
+In this walkthrough we'll show you how to allow your application to connect to external API outside of your mesh. This walkthrough will be a simplified version of the [Color App Example](https://github.com/aws/aws-app-mesh-examples/tree/main/examples/apps/colorapp).
+
+## Introduction
+
+When using a service mesh, some services within the mesh might need to connect to external or open API's.
+
+In App Mesh, we have two ways of doing that.
+
+### 1. Set egress filter to ``ALLOW_ALL``
+
+The first option is to set the [egress filter](https://docs.aws.amazon.com/app-mesh/latest/APIReference/API_EgressFilter.html) on the mesh resource to ``ALLOW_ALL``. This setting will allow any service within the mesh to communicate with any destination IP address inside or outside of the mesh.
+
+### 2. Model the external service as a virtual service backed up by a virtaul node
```

Comment:
in this heading can you include ``DROP_ALL`` so that we know that we are addressing DROP_ALL condition as well ?

Also feel free to include mesh definition to that the intention is more clear. Paragraph may not easily capture our attention by code block will easily do.

```
  Mesh:
    Type: AWS::AppMesh::Mesh
    Properties:
      MeshName: !Ref MeshName
      Spec:
        EgressFilter:
          Type: DROP_ALL
```